.github/CONTRIBUTING.md

```diff
@@ -1,16 +1,14 @@
 ## Contributing
 
-Please join us on [chat](https://chat.awesh.unknownplus.com/channel/awesome-selfhosted) to clarify any questions.
+Please join us on [chat](https://chat.awesh.unknownplus.com/channel/awesome-selfhosted) or open a new issue to clarify any questions.
```

Comment:
Should this link be included any more? The certificate has expired back in November.